README.md

```diff
@@ -10,6 +10,110 @@ Centos 6
 
 Centos 7?
 
+## Usage
+
+### Resource
+
+```
+    python_webapp "python_app_name" do
+      option "option_value"
+    end
+```
+
+Option                  | Type      | Description                                   | Default Value
+----------------------- | --------- | --------------------------------------------- | -------------
+`:create_user`          | Boolean   | Creates a user on the sever                   | `false`
+`:path`                 | String/Nil| Path to application                           | `nil` with a calculated default.
+`:owner`                | String    | Owner of application files (user)             | `chef`
+`:group`                | String    | Owner of application files (group)            | `chef`
+`:repository`           | String    | URL to git repository                         | `cannot be nil`
+`:revision`             | String    | Branch or Commit Hash of Repo                 | `master`
+`:virtualenv_path`      | String    | Path to python virtualenv                     | `nil` with a calculated default.
+`:config_template`      | String/Nil| Path to app config template                   | `nil` with a calculated default.
+`:config_destination`   | String/Nil| Path to app config location on server         | `nil` with a calculated default.
+`:config_vars`          | Hash      | Variables for the config template             | `Cannot be nil if config_destination is set, can be empty`
```

Comment:
Does it default to `nil`? If so, a default of `{}` might make more sense. Or some logic in the provider to use `{}` if it is set to nil.